Gosu/GraphicsBase.hpp

```diff
@@ -26,7 +26,9 @@ namespace Gosu
         //! to the old color's channels.
         AM_ADD,
         //! The color's channels will be multiplied with each other.
-        AM_MULTIPLY
+        AM_MULTIPLY,
+        //! The color will completly replace the current one.
```

Comment:
*completely

What is this mode good for? I've thought about extending the AlphaMode enum to cover all of [Adobe's blend modes](https://helpx.adobe.com/photoshop/using/blending-modes.html) because I'm familiar with them from Photoshop, but I'm not sure what `AM_REPLACE` could be used for.

Reply:
(I can think of use cases when drawing into an RGBA buffer; `Gosu::Bitmap::insert` is basically `AM_REPLACE`. But when you are drawing to the screen, you can't replace the destination pixels with translucent pixels.)